Sources/Server/Server.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import Socket
+
+public class Server {
+    
+    private let port: Int
+    private let acceptNewConnection = true
+    static let body = "<!DOCTYPE html><html><body><h1>Hello World</h1></body></html>"
+    static let header = "HTTP/1.1 200 OK\r\nContent-Length: \(body.utf8.count)\r\nContent-type: text/html\r\n\r\n"
+    
+    public init(port: Int){
+        self.port = port
+    }
+
+    public func startServer() throws {
+        let socket = try Socket.create()
+        try socket.listen(on: port)
+        repeat {
+            let connectedSocket = try socket.acceptClientConnection()
+            let request = try connectedSocket.readString()! 
+            let httpRequest = parseRequest(request: request)
+            let response = prepareResponse(request: httpRequest)
+            try connectedSocket.write(from: response)
+            connectedSocket.close()
+        } while acceptNewConnection
+    }
+
+    public func parseRequest(request: String) -> String {
```

Comment:
This method's name might be a little too 'forward thinking' perhaps for now it could be called getRequestMethod?